crates/shared/src/arguments.rs

```diff
@@ -147,6 +147,12 @@ pub fn parse_fee_factor(s: &str) -> Result<f64> {
     Ok(fee)
 }
 
+pub fn parse_unwrap_factor(s: &str) -> Result<f64> {
```

Comment:
nit: My initial thought was "why don't we reuse the same parsing as for the fee factor?" (answer, because this one here is upper bound by 1).

Maybe we can rename these methods to be more explanatory on the kind of parsing they do (e.g. `parse_percentage_factor` vs `parse_unbounded_percentage_factor`) rather than which field they are used for (I would like them to be reused if we add another field)?

Reply:
Good suggestions, will do.